src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -154,9 +154,7 @@ def _uca_bulk_record(
         if db_write_keys.size == 0:
             return db_read_key_results
 
-        config = get_ingest_config(
-            self._get_metric_path_key(strings.keys()), IndexerStorage.POSTGRES
-        )
+        config = get_ingest_config(self._metric_key_path, IndexerStorage.POSTGRES)
```

Comment:
We might want to consider moving this to the constructor if config stays the same throughout the life time of this object, but maybe there's a reason that it's in `bulk_record`